schemas/sdd-plus-superpowers/schema.yaml

```
@@ -0,0 +1,282 @@
+name: sdd-plus-superpowers
```

YAMLlint (1.38.0)

[error] wrong new line character: expected \n

(new-lines)